applications/PfemApplication/custom_bounding/spatial_bounding_box.hpp

```diff
@@ -852,39 +860,35 @@ class KRATOS_API(PFEM_APPLICATION) SpatialBoundingBox
 	start=1;
```

Comment:
Hi, I think these four lines (from unsigned int start = 0 to start=1) may be removed since the variable start is not used and creates a compiler warning. KR

Reply:
It is possible mistake. I have not detected it. Please can you add the correction to the branch?

Reply:
I think I have done it ;-P